x/evm/keeper/querier_test.go

```diff
@@ -0,0 +1,106 @@
+package keeper_test
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/cosmos/ethermint/app"
+	"github.com/cosmos/ethermint/x/evm"
+	"github.com/cosmos/ethermint/x/evm/keeper"
+	"github.com/cosmos/ethermint/x/evm/types"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+type QuerierTestSuite struct {
+	suite.Suite
+
+	ctx     sdk.Context
+	querier sdk.Querier
+	app     *app.EthermintApp
+	codec   *codec.Codec
+}
+
+func (suite *QuerierTestSuite) SetupTest() {
+	checkTx := false
+
+	suite.app = app.Setup(checkTx)
+	suite.ctx = suite.app.BaseApp.NewContext(checkTx, abci.Header{Height: 1, ChainID: "3", Time: time.Now().UTC()})
+	suite.querier = keeper.NewQuerier(suite.app.EvmKeeper)
+	suite.codec = codec.New()
+}
+
+func TestQuerierTestSuite(t *testing.T) {
+	suite.Run(t, new(QuerierTestSuite))
+}
```

Comment:
you can use the same `KeeperTestSuite` and add a `querier sdk.Querier` field as you did above.

Reply:
see below